### PR TITLE
portaudio: Update to 190600_20161030

### PR DESCRIPTION
**Pull request by neheb**
Signed-off-by: Rosen Penev <rosenp@gmail.com>

Maintainer: @psycho-nico 
Compile tested: ipq806x

### CODE REVIEW

**Comment by diizzyy** [comment]
Given that the only user is espeak and it's dead upstream it's perhaps time to retire these packages?
https://sourceforge.net/projects/espeak/files/espeak/

**Comment by neheb** [comment]
I have no issue with such a thing.

**Comment by diizzyy** [comment]
Let's revive this discussion again, Nicolas hasn't replied for several months on queries about packages so I think we can by now assume that he's not available/lost interest. The only user is espeak which doesn't even build.

http://downloads.lede-project.org/snapshots/faillogs/mipsel_24kc/packages/espeak/compile.txt

Let's nuke portaudio and espeak (more information here)
https://github.com/openwrt/packages/pull/4583

@thess @hnyman

**Comment by diizzyy** [comment]
@dibdot @karlp @pprindeville 
Reasonable course of action?

**Comment by neheb** [comment]
I'm normally against putting packages in some abandoned repo, but I vote for nuking this.

File: sound/portaudio/Makefile [hunk 1]
```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=portaudio
-PKG_VERSION:=19_20140130
-PKG_RELEASE:=2
+PKG_VERSION:=190600_20161030
```
Comment (pprindeville) [inline sound/portaudio/Makefile]: Why are we changing the versioning?

**Comment by diizzyy** [comment]
I'm not sure why we even need to bother as long as we keep the git history (commit msg) sane?

File: sound/portaudio/Makefile (hunk 1 above)
Comment (neheb) [reply sound/portaudio/Makefile]: Because that's the version of the package

**Comment by pprindeville** [comment]
There are ways to copy the git history into a new repo.

We'd do so mostly to stop anyone from building this as it's no longer safe.

**Comment by diizzyy** [comment]
Huh? I'm pretty sure if the commit msg says that it's being removed for deprecation or whatever reason it's pretty clear that you're on your own doing otherwise.

**Comment by pprindeville** [comment]
Not everyone bothers to read the git logs.

**Comment by thess** [comment]
Superseded by #6908

**Pull request closed without merging**